src/chrome/__mocks__/chrome.js

```diff
@@ -230,6 +252,10 @@ export const mockChrome = (opts = {}) => {
 
   const getOpenTabs = () => openTabs;
 
+  const closeWindow = () => {
+    openWindows = [];
+  };
```

Comment:
Or actually I'll just rename to `closeWindows` for clarity (it's not a real chrome method).